Dockerfile

```diff
@@ -3,18 +3,14 @@ FROM debian:stable-slim
 RUN apt-get clean && apt-get update && \
     apt-get install -y wget gcc gcc-avr avr-libc gnupg
```

Comment:
I think `gnupg` is not necessary anymore since `apt-key` isn't used.

Reply:
OK also removed.